cmake/DaemonFlags.cmake

```diff
@@ -185,26 +185,49 @@ else()
     set_c_cxx_flag("-ffast-math")
     set_c_cxx_flag("-fno-strict-aliasing")
 
-    # Set arch on x86 to SSE2 minimum and enable CMPXCHG16B
-    if (ARCH STREQUAL "i686")
-        set_c_cxx_flag("-m32")
-        set_c_cxx_flag("-msse2")
-        set_c_cxx_flag("-mtune=generic")
-        try_c_cxx_flag_werror(MFPMATH_SSE "-mfpmath=sse")
-    elseif (ARCH STREQUAL "amd64")
-        set_c_cxx_flag("-m64")
-        set_c_cxx_flag("-mtune=generic")
-        try_c_cxx_flag_werror(MCX16 "-mcx16")
+    if (ARCH STREQUAL "amd64")
+        set(GENERIC_ARCH "x86-64")
```

Comment:
I suggest naming this variable GCC_GENERIC_ARCH, to make it clear that the name is just used for feeding GCC flags

Reply:
done.